imap_processing/tests/test_decom.py

```diff
@@ -0,0 +1,61 @@
+"""Tests aspects of the decommutation process that is generic to all instruments"""
+
+import glob
+
+import pytest
+
+from imap_processing import imap_module_directory
+
+xtce_documents = glob.glob(f"{imap_module_directory}/*/packet_definitions/*.xml")
+xtce_documents = [
+    item for item in xtce_documents if "idex_packet_definition.xml" not in item
+]  # Temporary until IDEX packet definition is fixed
+
+
+@pytest.mark.parametrize("xtce_document", xtce_documents)
+def test_ccsds_header(xtce_document):
+    """Test if the XTCE document contains the proper CCSDS header information"""
+
+    print(f"\nTesting {xtce_document}")
+
+    # Define what is expected in the XTCE document
+    header_keys = [
+        "VERSION",
+        "TYPE",
+        "SEC_HDR_FLG",
+        "PKT_APID",
+        "SEG_FLGS",
+        "SRC_SEQ_CTR",
+        "PKT_LEN",
+    ]
+    header_container_set = (
```

Comment:
Related to above, header_container_set does include SHCOARSE while the above code doesn't. Do we want those to match? Maybe we can add a check for MET _or_ SHCOARSE?

Reply:
Yes, these should match, and I will update accordingly depending on if we decide that `SHCOARSE` is part of the header or not.

I am inclined to say it doesn't belong in the header, because that is also how [`space_packet_parser` defines it](https://github.com/medley56/space_packet_parser/blob/bef50e6bfd262fdef4727381c5dafea6488571c2/space_packet_parser/parser.py#L20-L27).

Reply:
This was resolved by #185